setup.py

```diff
@@ -44,7 +44,7 @@ def readfile(name):
 
 
 docs_extras = [
-    'Sphinx >= 1.8.1',  # Unicode characters in tree diagrams
+    'Sphinx == 1.8.5',  # Current version for Read the Docs
```

Comment:
Is this pin necessary? RTD will install Sphinx 1.8.5 without it. We want to upgrade to 2.x whenever RTD gets around to it. They rarely make such an announcement. It's better to have the flexibility to build with either 1.8.5 on RTD or 2.0 locally (and whenever RTD upgrades).

Reply:
Happy to revert it if you want.

Reply:
Yes, please. Then we can forget about it.

Here's the requirement:
https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt#L13

Reply:
Reverted.

Ignorant question:  Could we put `sphinx >= 2.0.0` in the doc extras to force RTD to use 2.0?

Reply:
_Good_ question! Nope, I tried it a while ago. They pin Sphinx to avoid breaking customer docs.